style.css

```diff
@@ -7,7 +7,7 @@ Description: WordPress Design System theme is a theme that leverages the full-si
 Requires at least: 6.4.4
 Tested up to: 6.5
 Requires PHP: 7.4
-Version: 1.4.0
+Version: 1.5.0
```

Comment:
If your changing the version, don't forget to update the composer.json file if you haven't already done so.